google-authenticator.php

```diff
@@ -217,6 +217,9 @@ function user_needs_to_setup_google_authenticator() {
 	}
 
 	$must_signup = false;
+	if ( ! count( $user->roles ) ) {
+		return false;
```

Comment:
Odd, albeit valid use case.

I'd still suggest consistency with the filter and do something like:
`	return apply_filters( 'google_authenticator_needs_setup', $must_signup, $user );
`
instead of return false.